cwms-data-api/src/main/java/cwms/cda/api/timeseriesprofile/TimeSeriesProfileInstanceController.java

```diff
@@ -315,7 +139,7 @@ public void getOne(@NotNull Context ctx, @NotNull String timeSeriesId) {
                     .getOrDefault(false);
             String page = ctx.queryParam(PAGE);
             int pageSize = ctx.queryParamAsClass(PAGE_SIZE, Integer.class).getOrDefault(500);
-            CwmsId tspIdentifier = new CwmsId.Builder().withOfficeId(officeId).withName(timeSeriesId).build();
+            CwmsId tspIdentifier = new CwmsId.Builder().withOfficeId(officeId).withName(locationId).build();
```

Comment:
the identifier for a time series profile instance is the parameter + location + version string right?

There's also the optional date version that's getting lost in this CwmsId.

Reply:
The identifier is parameter + location + version + version date.

I renamed this variable to tspLocationIdentifier to clarify what it is used for.

Reply:
ah gotcha, that's my misunderstanding. location-id is much better and more consistent